lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -2872,7 +2884,8 @@ def permutation_matrices(n):
             qinds = np.arange(qc)
             rinds = np.arange(rc)
 
-            square_rot = square.dot(permute.T)
+            square_rot_pos = square.dot(permute.T)
```

Comment:
```suggestion
            square_rot_pos = square @ permute.T
```
We can use the matrix multiplication operator now that we are on Py3.

Reply:
Not a bad idea - although your suggested change doesn't let me change both at once through the web.

Has matplotlib already switched all uses of `.dot` to `@` internally? If not, I'd be inclined to declare this out of scope, and leave it for later cleanup.

Reply:
No, we have not. You may leave it for later if you wish to.

Reply:
Will change if another revision is justified for another reason.

Reply:
Actually, I'd like to leave that to an axes3d-wide cleanup of `dot`